staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd.go

```diff
@@ -155,10 +158,14 @@ func (r *ScaleREST) Update(ctx genericapirequest.Context, name string, objInfo r
 		return nil, false, err
 	}
 
-	oldScale, err := scaleFromCustomResource(cr, r.specReplicasPath, r.statusReplicasPath, r.labelSelectorPath)
+	const invalidSpecReplicas = -2147483648 // smallest int32
```

Comment:
@deads2k @DirectXMan12 any better idea than this? We have to exclude patches that do not set the missing replicas field.

Reply:
> @deads2k @DirectXMan12 any better idea than this? We have to exclude patches that do not set the missing replicas field.

Nasty.  I can live with it if need be.